spec/system/candidate_interface/candidate_needs_to_provide_two_new_referees_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate needs to provide 2 new referees' do
+  include CandidateHelper
+
+  scenario "Candidate provides a new referee because 2 didn't respond" do
+    FeatureFlag.activate('show_new_referee_needed')
+
+    given_i_am_signed_in_as_a_candidate
+    and_i_have_submitted_my_application
+    and_both_of_my_referees_havent_responded_within_10_days
+
+    when_i_visit_the_application_dashboard
+    then_i_see_that_i_need_new_references
+
+    when_i_click_to_add_a_new_referee
+    and_i_fill_in_the_form
+    and_i_fill_in_the_second_form
+    then_i_see_the_reference_review_page
+
+    when_i_click_to_edit_the_referee
+    and_i_edit_the_name
+    then_i_see_the_updated_name_on_the_confirm_page
+
+    when_i_click_to_confirm
+    then_the_new_referees_should_receive_emails
+    and_i_see_that_my_new_references_have_been_requested
+    and_i_should_not_be_asked_to_provide_another_reference
+
+    when_i_go_back_to_the_edit_page
+    then_i_see_a_404_page
+  end
+
+  def given_i_am_signed_in_as_a_candidate
+    @candidate = create(:candidate)
+    login_as(@candidate)
+  end
+
+  def and_i_have_submitted_my_application
+    @application_form = create(:completed_application_form, candidate: @candidate)
+  end
+
+  def and_both_of_my_referees_havent_responded_within_10_days
+    create(:reference, :requested, application_form: @application_form, created_at: Time.zone.now - 14.days)
```

Comment:
14 days versus 10 days?

Reply:
Updated (made it 30 days and updated the step definition)